module-2/backend/week_two.md

```diff
@@ -13,8 +15,21 @@ end
 ```
 
 What are some methods you can call on `Team`? If these methods aren't defined in the class, how do you have access to them?
+I can call find, group, select, where. 
+If the methods aren't defined in the class then I am not sure as how we have access to them.
 
 3. Assume that in your database, a team has the following attributes: "id", "name", owner_id". How would you find the name of a team with an id of 4? Assuming your class only included the code from question 2, how could you find the owner of the same team?
+ ```ruby
+  team4 = Team.find(4)
+  team4.name
+end
+```
+
+```ruby
+  team4 = Team.find(4)
+  team4.owner_id
+  end
```

Comment:
Same as above.

Also, this finds the owner id, but not the owner itself. In order to find the owner, we would need to do something like the following:

```ruby
owner_4_id = team4.owner_id
Owner.find(owner_4_id)
```